Source/WebKit/Platform/cocoa/AssertionCapability.mm

```diff
@@ -46,13 +46,7 @@
 ALLOW_DEPRECATED_DECLARATIONS_END
 {
 #if USE(EXTENSIONKIT)
-#if USE(EXTENSIONKIT_INVALIDATION_CALLBACKS)
     return [get_SECapabilitiesClass() assertionWithDomain:m_domain name:m_name environmentIdentifier:environmentIdentifier() willInvalidate:m_willInvalidateBlock.get() didInvalidate:m_didInvalidateBlock.get()];
-#else
-    if ([get_SECapabilitiesClass() respondsToSelector:@selector(assertionWithDomain:name:environmentIdentifier:)])
-        return [get_SECapabilitiesClass() assertionWithDomain:m_domain name:m_name environmentIdentifier:environmentIdentifier()];
```

Comment:
Maybe you should keep the respondsToSelector check until we know all build and test bots have this method? Or has this API been around for a long time?

Reply:
Fixed!

Thanks for reviewing!